src/main/java/com/passbook/sparkeighteen/peristence/entity/UserEntity.java

```diff
@@ -1,32 +1,56 @@
 package com.passbook.sparkeighteen.peristence.entity;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.passbook.sparkeighteen.peristence.POJO.Gender;
 import lombok.*;
-import org.hibernate.annotations.GenericGenerator;
+import org.springframework.format.annotation.DateTimeFormat;
 
 import javax.persistence.*;
-import java.util.UUID;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.time.LocalDate;
 
-@Data
+@Setter
+@Getter
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @Entity
-@Table(name = "user",schema = "public")
+@Table(name = "user", schema = "public")
 public class UserEntity {
 
+
     @Id
-    @GeneratedValue(generator = "UUID")
-    @GenericGenerator(
-            name = "UUID",
-            strategy = "org.hibernate.id.UUIDGenerator"
-    )
+    @GeneratedValue(strategy = GenerationType.AUTO)
     @Column(name = "id")
-    private UUID userId;
+    private Integer id;
 
-    @Column(name = "email")
-    private String email;
+    @NotNull
+    @Column(name = "firstname")
+    private String firstname;
+
+    @Column(name = "lastname")
+    private String lastname;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "gender", length = 20)
+    private Gender gender;
+
+    @NotNull
+    @Column(name = "dob")
+    @DateTimeFormat(pattern = "yyyy-MM-dd")
+    private LocalDate dob;
 
     @Column(name = "password")
     private String password;
 
+    @NotNull
+    @Column(name = "email")
+    @Email(message = "Email is not valid", regexp = "[a-z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,3}", flags = Pattern.Flag.CASE_INSENSITIVE)
```

Comment:
This Is wrong.
See the coffee code of google more precisely

Reply:
@NotNull
    @Pattern(regexp = "^(.+)@(.+)$")
    @Column(name = "email")
    @Email(message = "Email")
    private String email;